packages/mobile/src/components/premium-track-purchase-drawer/PremiumTrackPurchaseDrawer.tsx

```diff
@@ -202,33 +206,38 @@ const RenderForm = ({ track }: { track: PurchasableTrackMetadata }) => {
   return (
     <>
       <ScrollView contentContainerStyle={styles.formContentContainer}>
-        <TrackDetailsTile trackId={track.track_id} />
-        {isPurchaseSuccessful ? null : (
-          <PayExtraFormSection amountPresets={payExtraAmountPresetValues} />
-        )}
-        <PurchaseSummaryTable
-          {...purchaseSummaryValues}
-          isPurchaseSuccessful={isPurchaseSuccessful}
-        />
-        {isPurchaseSuccessful ? (
-          <PurchaseSuccess track={track} />
-        ) : (
-          <View>
-            <View style={styles.payToUnlockTitleContainer}>
-              <Text weight='heavy' textTransform='uppercase' fontSize='small'>
-                {messages.payToUnlock}
+        <View style={styles.formContentSection}>
+          <TrackDetailsTile trackId={track.track_id} />
+        </View>
+        <AudioMatchSection amount={Math.round(price / 100)} />
```

Comment:
So are we always showing the earn audio banner, even after purchase?

Reply:
ahhh thank you for this